Tracing Demo DRAFT (#11518)
getsentry/sentry-docs: docs/product/sentry-basics/distributed-tracing/initialize-sentry-sdk-backend.mdx

   const { nodeProfilingIntegration } = require("@sentry/profiling-node");

   Sentry.init({
    dsn: "sample_DSN_key",
    integrations: [
        nodeProfilingIntegration(),
    ],
   // Tracing
   tracesSampleRate: 1.0,
   //  Capture 100% of the transactions

   // Set sampling rate for profiling - this is relative to tracesSampleRate
   profilesSampleRate: 1.0,
   });
   ```

   It's important to import and initialize the SDK as early as possible in your application's lifecycle so Sentry can capture errors throughout the lifecycle.

1. Add your DSN key to the Sentry SDK configuration.

   Paste in the DSN key value you copied from the project created in the [previous section](/product/sentry-basics/distributed-tracing/create-new-project/).
   {/* TODO: Update link here */}

   ```javascript {filename:instrument.js}
   Sentry.init({
     dsn: "<your_DSN_key>",
     // ...
   });
   ```

1. Save the file.

The options set in `Sentry.init()` are called the SDK's configuration. The only required configuration option is the DSN. However, the SDK supports many other configuration options. Learn more in our [Configuration](/platforms/javascript/guides/react/configuration/) docs.

> The configuration above enables Sentry's error monitoring feature, as well as its [Performance](/platforms/javascript/guides/react/performance) (tracing) and [Session Replay](/platforms/javascript/guides/react/session-replay) features.
> 

Reviewer: ```suggestion
The configuration above enables Sentry's error monitoring feature, as well as its [Tracing](will add link later) and [Session Replay](/platforms/javascript/guides/react/session-replay) features.

```

> Make sure to import `instrument.js` at the top of your `server.js` file, as well as importing `Sentry` and setting up the error handler after all controllers and before any other middleware:

Reviewer: ```suggestion
Be sure to import `instrument.js` at the top of your `server.js` file. Then import `Sentry` and set up the error handler after all controllers and before any other middleware:
```
